tstest/integration/vms/vms_test.go

```diff
@@ -280,6 +282,7 @@ func mkVM(t *testing.T, n int, d Distro, sshKey, hostURL, tdir string) func() {
 		"-drive", driveArg,
 		"-cdrom", filepath.Join(tdir, d.name, "seed", "seed.iso"),
 		"-vnc", fmt.Sprintf(":%d", n),
+		"-smbios", "type=1,serial=ds=nocloud;h=" + d.name,
```

Comment:
Why this?

Reply:
The [NoCloud](https://cloudinit.readthedocs.io/en/latest/topics/datasources/nocloud.html) datastore really prefers having a SMBIOS variable telling it where to look for data. In this case we are just telling it to read from the cloud-init seed, however when I implement the network metadata storage part of #1988 it will be how we tell where cloud-init should read metadata from. This was also an ill-fated attempt to get OpenSUSE 15.1 working (my VM puppeting setup that uses the same basic stack as this testsuite is able to work with it, and it sets that smbios variable, however it's just not working and IDK why; I'll figure that out later).

tl;dr: It's all about correctness in how I do satanic things with cloud-init.